tests/integration/ha/test_small_deployment_upgrades.py

```diff
@@ -154,6 +130,10 @@ async def test_upgrade_between_versions(
     leader_id = [u.id for u in units if u.is_leader][0]
 
     for version, rev in VERSION_TO_REVISION.items():
+        if version == FIRST_VERSION:
```

Comment:
Can we please rename `FIRST_VERSION` to something along the lines of: `CURRENT_VERSION_MINUS_2` or similar?

Reply:
@Mehdi-Bendriss can we rename to `STARTING_VERSION` ? I don't want to pin we will only test down to `N-2` here, but rather on the `VERSION_TO_REVISION` dict.